{
  "timestamp": 1736620718036,
  "repository": "Bug-Hunter-X/Uncommon-Dockerfile-Error--Using--ubuntu-latest--and-Inconsistent-Paths-zeizw",
  "path": "Dockerfile",
  "category": "docker",
  "stars": 0,
  "content": "FROM ubuntu:latest\nRUN apt-get update && apt-get install -y python3 python3-pip\nCOPY . /app\nWORKDIR /app\nCMD [\"python3\", \"/app/main.py\"]"
}